src/platform/site-wide/sass/iconography.scss

```diff
@@ -126,6 +132,7 @@
   $hub-name: "life-insurance",
   $icon: $fa-var-shield-alt,
   $color: $green-warm-50,
+  $icon-weight: 900,
```

Comment:
`900` indicated to use the solid icon set of FA Free, while 400 is the regular set.